_includes/doc/admin-guide/options/source-flags.md

```diff
@@ -139,5 +139,6 @@
     If the BOM[^1] character is missing, but the message is otherwise UTF-8
     compliant, {{ site.product.short_name }} automatically adds the BOM character to the
     message.
+- *no-piggyback-errors*: This option configures syslog-ng OSE to ignore parsing errors caused by piggybacked messages. If the parser encounters issues separating multiple messages sent together, it skips them without raising an error.
```

Comment:
This description does not make much sense to me.

Reply:
@zsoltgyulai94 could you please check [this](https://github.com/syslog-ng/syslog-ng/pull/5151/files) PR description and rephrase based on it, TIA!

Reply:
Yeah, sorry, wasn't aware of that PR. Fixed.